client/src/interface/Contest.ts

```diff
@@ -1,13 +1,20 @@
+import { User } from './User';
+
 export interface Contest {
   title: string;
   description: string;
   prizeAmount: number;
   deadlineDate: string;
-  userID: string;
+  userID?: User;
```

Comment:
Okay ill change that

Reply:
I tried changing to from userID to user but i kept getting undefined. I guess it's because that's how we assigned the model on the server.  

![Screenshot (28)](https://user-images.githubusercontent.com/33403043/127725431-936a0b8d-50b0-42e1-81fb-a9fbf097721d.png)

Reply:
Because we assigned userID as the model the data returs userID. if you have a way around it let me know